### PR TITLE
Seed summary modifications

### PR DESCRIPTION
**Pull request by giladt**
- [Fixed the padding for `.summaryContainer`](https://www.notion.so/curvelabs/Seed-Summary-Increase-the-padding-between-sections-7cfbda2d338f40e1bde898a81ee61ba2)

- [Modified text changes](https://www.notion.so/curvelabs/Seed-Summary-Text-Changes-15b657f77f0b49408dc963d2a2501a2a)

### CODE REVIEW

File: src/newSeed/baseStage.scss [hunk 1]
```diff
@@ -539,7 +539,12 @@
             background-color: $Shade03;
           }
         }
+
+        + * + * {
```
Comment (dkent600) [inline src/newSeed/baseStage.scss:543]: what does this selector do?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/baseStage.scss (hunk 1 above)
Comment (giladt) [reply src/newSeed/baseStage.scss:543]: Same as `:not(:first-child)` but shorter...

Comment (dkent600) [reply src/newSeed/baseStage.scss:543]: shorter is not always better.  can we please use the long version since it is very clear in meaning?

Comment (giladt) [reply src/newSeed/baseStage.scss]: done.

**Review by dkent600 — APPROVED** [review]

**Pull request merged**